hugo.toml

```
@@ -1,5 +1,5 @@
 baseURL = 'https://googlecloudplatform.github.io/samples-style-guide/'
-title = 'Samples Style Guide | Google Cloud'
+title = 'Samples Guide | Google Cloud'
```

This change was related to other changes in my local copy, only part of it doesn't make sense.

```suggestion
title = 'Samples Style Guide | Google Cloud'
```